apps/web/src/components/layout/components/v2/SyncInfoModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { Modal, Title, colors } from '@novu/design-system';
+import { Button, Code } from '@novu/novui';
+import { Tabs, Text } from '@novu/novui';
+
+import { IconOutlineCloudUpload, IconPencil, IconLink } from '@novu/novui/icons';
+import { useState } from 'react';
+import { useEnvironment } from '../../../../hooks/useEnvironment';
+import { useApiKeysPage } from '../../../../pages/settings/ApiKeysPage/useApiKeysPage';
+import { getBridgeUrl } from './utils';
+// TODO: make code copy paste
```

Comment:
The content of the code block is copy pastable, but a convenience button might be wanted as well